src/login/loginctl.c

```diff
@@ -144,49 +144,85 @@ static int show_table(Table *table, const char *word) {
         return 0;
 }
 
-static int list_sessions(int argc, char *argv[], void *userdata) {
+static int list_sessions_table_add(Table *table, sd_bus_message *reply) {
+        int r;
+
+        assert(table);
+        assert(reply);
+
+        r = sd_bus_message_enter_container(reply, 'a', "(sussussbto)");
+        if (r < 0)
+                return bus_log_parse_error(r);
+
+        for (;;) {
+                const char *session_id, *user, *seat, *class, *tty;
+                uint32_t uid, leader_pid;
+                bool idle;
+                usec_t idle_timestamp_monotonic;
```

Comment:
might make sense to change this type to uint64_t, even if it's the same as usec_t, since that's what sd_bus_message_read() actually wants...